webapp/src/app/api/translations/[projectName]/[lang]/[msgId]/route.ts

```diff
@@ -23,11 +23,10 @@ export async function PUT(
   const { lang, msgId, projectName } = context.params;
   const payload = await req.json();
   const { text } = payload;
-  // TODO: include getProjectConfig & readFromDir in a try/catch block and check for error to return a certain 500 error
+  // TODO: include getProjectConfig() in a try/catch block and check for error to return a certain 500 error
```

Comment:
in the main branch `readFromDir()` is called below this line, now since we are not calling it from here anymore the TODO should be about `getProjectConfig()`, but now I see your point, I will add `getLyraConfig()` to the todo